pkg/registry/core/backupbucket/strategy.go

```diff
@@ -131,7 +131,6 @@ func (backupBucketStatusStrategy) ValidateUpdate(ctx context.Context, obj, old r
 }
 
 // ToSelectableFields returns a field set that represents the object
-// TODO: fields are not labels, and the validation rules for them do not apply.
```

Comment:
I didn't get what this todo was about, can you please explain?
other than that. LGTM!

Reply:
I don't know, I already cleaned some of them up as part of https://github.com/gardener/gardener/pull/7868/commits/71d58b9bb5462b540855a1a7cef880af66d14da9 in https://github.com/gardener/gardener/pull/7868. They existed since many years and nobody cared about them, so probably they are not relevant.

Reply:
Ok 👍.